contracts/vaults/NonvotingVault.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "../libraries/Authorizable.sol";
+import "../interfaces/ILockingVault.sol";
+import "../interfaces/IERC20.sol";
+
+contract NonvotingVault is Authorizable {
```

Comment:
The naming of the contract is not following the camelCase convention of solidity.